app/assets/javascripts/snowcrash/modules/notifications.js.coffee

```diff
@@ -0,0 +1,39 @@
+document.addEventListener 'turbolinks:load', ->
+  if $('.js-notifications-dropdown').length
+    $container = $('.js-notification-container')
+    $empty     = $container.find('.js-no-content')
+    $footer    = $container.next('.js-footer')
+    $loading   = $container.find('.js-loading')
+
+    $('.js-notifications-dropdown').click ->
+      unless $container.is(':visible')
+        $.ajax $container.data('url'),
+          dataType: 'json',
+          data: {count: 7},
+          beforeSend: () ->
+            $container.find('li.notification').remove()
+
+            $empty.hide()
+            $footer.hide()
+            $loading.show()
+
+          success: (data) ->
+            $('.js-notification-count').html("Notifications (#{data.count})")
+
+            if data.count == 0
+              $empty.show()
+            else
+              for notification in data.notifications
```

Comment:
We may build the html server side.
Using here just a `remote: true` link + `format :js` on the controller.

Thoughts?